jax/experimental/jax2tf/jax2tf.py

```diff
@@ -107,9 +107,20 @@ def _safe_convert_to_tensor(val, dtype=None) -> TfVal:
 tf_impl_with_avals: Dict[core.Primitive,
                          Callable[..., Any]] = {}
 
+# XLA is not linked in all environments; when converting a primitive, if the
+# conversion path involves a TFXLA operation and this variable is disabled, an
+# exception is thrown instead.
+ENABLE_XLA = True
```

Comment:
I would suggest to name this "_enable_xla", with "_" to mark it private, and with lower case because it is a variable.